src/modules/vendor/components/courses/CourseCard.vue

```diff
@@ -48,6 +51,43 @@ export default {
         { icon: 'emoji_people', label: `${this.trainerName}` },
       ];
     },
+    formatNearestDate () {
+      if (this.courseSlotsCount === 0) {
+        return { label: 'Pas de date prévue', icon: 'img:statics/calendar-arrow-right.svg' };
+      }
+      const listDaySlots = Object.values(this.course.slots);
+
+      if (listDaySlots.every((daySlot) => !this.isDone(daySlot))) {
+        const firstSlot = listDaySlots[0];
+        const rangeToNextDate = this.$moment(firstSlot[0].startDate).diff(this.$moment(0, 'HH'), 'd');
```

Comment:
Tu peux utiliser plutot `this.$moment.startOf('day')` a la place de `this.$moment(0, 'HH')`